artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/PacketImpl.java

```diff
@@ -31,7 +31,8 @@
 
    // 2.0.0
    public static final int ADDRESSING_CHANGE_VERSION = 129;
-   public static final int SHARED_QUEUE_SECURITY_FIX_CHANGE_VERSION = 130;
```

Comment:
This was used in version 2.6.1 shouldn't be removed, need to bump the async response change version. (as was made prior to the security fix version)